### DISCUSSION TITLE
Invalid format

### DISCUSSION THREAD

**Discussion by jarrah42** [silx-kit/h5web#1196] [Q&A] [answered]
Hi,

I'm trying to use @h5web/app as a visualization tool for Galaxy (https://galayproject.org). It looks like Galaxy provides an iframe for the tool, and I'm able to get the App loading using the MockProvider. Everything seems to work ok except the HeatMap, which displays the legend for an instant and then displays "invalid format: .3~g" in red. The remaining controls are missing, so I'm guessing this exception stops them being drawn. Every dataset does the same thing, with the same error.

Do you have an inkling of what might be causing this. I've tried looking for this error in the source by to no avail, so I'm guessing it is deeper in one of the libraries somewhere.

It's a bit of a mystery why this would be working and everything else looks ok also.

Thanks!

**Comment by jarrah42** [comment, marked as answer]
Turns out I was using a really old version of d3-format. Working now!

**Reply by loichuder** [reply]
Sorry just saw your issue now. Glad to see you could fix it :smile:

**Comment by axelboc** [comment]
Hi @jarrah42! We're very curious to know how you go about integrating H5Web into Galaxy. Once you have something useful working, would you be able to please share with us maybe a link to a small demo (if that's not too much work), or at least some screenshots?

**Reply by jarrah42** [reply]
Yes definitely!

**Comment by jarrah42** [comment]
I have submitted a pull request to galaxy if you want to take a look: https://github.com/galaxyproject/galaxy/pull/15394

**Reply by axelboc** [reply]
Awesome!!
